### PR TITLE
Fix typo from #65214

### PR DESCRIPTION
**Pull request by Amanieu**

### CODE REVIEW

File: src/libstd/panic.rs [hunk 1]
```diff
@@ -264,7 +264,7 @@ impl RefUnwindSafe for atomic::AtomicI128 {}
 #[cfg(target_has_atomic_load_store = "ptr")]
 #[stable(feature = "unwind_safe_atomic_refs", since = "1.14.0")]
 impl RefUnwindSafe for atomic::AtomicUsize {}
-#[cfg(target_hastarget_has_atomic_load_store_atomic = "8")]
+#[cfg(target_has_atomic_load_store_atomic = "8")]
```
Comment (jonas-schievink) [inline src/libstd/panic.rs]: Should this not be:

```suggestion
#[cfg(target_has_atomic_load_store = "8")]
```

Comment (Amanieu) [reply src/libstd/panic.rs]: Yes...

**Comment by Centril** [comment]
r? @jonas-schievink 

Apparently I cannot be trusted to review this reliably today :P

**Pull request merged**